v1/pipeline/steps/common/aws-cdk-bootstrap-step-info.yaml

```diff
@@ -52,6 +59,9 @@ allOf:
       - type: string
 $schema: http://json-schema.org/draft-07/schema#
 type: object
+required:
```

Comment:
will this not break existing pipelines

Reply:
there are no existing pipelines, feature is still under development.